src/main/java/uk/gov/hmcts/cmc/claimstore/events/ccd/CCDCaseHandler.java

```diff
@@ -49,7 +48,7 @@ public CCDCaseHandler(
         this.userService = userService;
     }
 
-    @TransactionalEventListener
```

Comment:
The async calls means that there's no guarantee that this call is executed before the document upload event if it's transactional

Reply:
Good catch!